k8s/releases/careers/careers-stage.yaml

```diff
@@ -0,0 +1,47 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: careers
```

Comment:
I tried to check that each of these that say 'careers' or 'careers-stage' are the correct one or the other, but I encourage a second look at that by someone else, in case I missed anything important.

Reply:
Ah, I see C did already.